docs/userguide/ui-concepts.md

```diff
@@ -1,3 +1,151 @@
 # Workflow UI
 
-_Coming soon!_
+This page provides a high level overview of Workflow UI, the companion that allows [Workflow Core](../core-workflow) to drive Android and iOS apps.
+
+## Separation of Concerns
+
+Workflow maintains a rigid separation between its core runtime and its UI support.
+The [Workflow Core](../core-workflow) modules are strictly Swift and Kotlin, with no dependencies on any UI framework.
+Those dependencies are restricted to the Workflow UI modules, as you would expect, with Swift UI relying on iOS, and Kotlin UI on Android.
+This innate separation naturally puts developers on a path to avoid entangling view concerns with their app logic.
+
+
+!!! info "Separate both ways"
+    This strictness goes in both directions: neither UI library depends upon either runtime. They could easily be driven by any other stream of view models, and in view system tests they often are.
```

Comment:
I'm not sure if we've honored this on the Swift side. We have an explicit [dependency](https://github.com/square/workflow-swift/blob/main/WorkflowUI.podspec#L21) on `Workflow`.

Reply:
Oh, that's too bad. Any idea why?

Reply:
Take this with a grain of salt as I'm still very new to the code base, but the dependency was introduced in 2de5e98c2 and it looks like it's primarily because ContainerViewController imports Workflow and references WorkflowHost, AnyWorkflowConvertible, etc.

Reply:
Thanks for the spelunking. I just deleted this block, it didn't seem important.